README.md

```diff
@@ -194,9 +194,11 @@ end.
 
 6. Run `make`. It should say `Looking good!` If one of the sanity checks fails,
    address the complaint and repeat. The various sanity checks are defined in
-   `common.mk`.
+   `common.mk`. The github credentials may be left blank.
```

Comment:
Since the prompt for Github credentials is dependent on how you cloned the repository and authenticate with Github, I would recommend changing this to something more general.

Perhaps, "Some of the checks make use of git, and if you cloned the repository with via http, you may be asked to authenticate. We recommend using ssh, but for the purposes of this check you can leave the credentials blank."

Feel free to change these words to your taste.